apps/ledger-live-mobile/src/screens/CustomImage/Step2Preview.tsx

```diff
@@ -169,43 +204,48 @@ const Step2Preview = ({ navigation, route }: NavigationProps) => {
         )}
       </Flex>
       <BottomButtonsContainer>
-        <Text fontSize="14px" lineHeight="17px">
-          {t("customImage.selectContrast")}
-        </Text>
         {resizedImage?.imageBase64DataUri && (
-          <Flex flexDirection="row" my={6} justifyContent="space-between">
-            {contrasts.map(({ val, color }) => (
+          <Flex flexDirection="row" my={6}>
+            <Animated.View style={leftBoxAnimatedStyle}>
+              <Box width={(3 - selectedIndex) * 54} />
```

Comment:
I don't think you need to add this <Box /> because the <Animated.View /> already has a width